packages/root-cms/ui/components/DocEditor/fields/DateTimeField.tsx

```diff
@@ -9,77 +9,127 @@ import {FieldProps} from './FieldProps.js';
 export function DateTimeField(props: FieldProps) {
   const field = props.field as DateTimeFieldSchema;
   const [value, setValue] = useDraftDocValue<Timestamp | null>(props.deepKey);
+
+  // Metadata is stored in a sibling field prefixed with `@`.
+  // e.g. if field is `scheduledAt`, metadata is in `@scheduledAt`.
+  const metadataKey = getMetadataKey(props.deepKey);
+  const [metadata, setMetadata] = useDraftDocValue<Record<string, any>>(
```

Comment:
For now since we only need the `timezone` metadata key, we should listen on only that key instead of the whole metadata object.